spec/integration/stub_requests_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "capybara"
```

Comment:
What do you reckon of splitting this out into a file that can be shared between all integration specs?